README.md

```diff
@@ -1,2 +1,103 @@
-# module-ballerinax-postgresql
-Ballerina PostgreSQL DB module
+Ballerina PostgreSQL Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerinax-postgresql/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-postgresql/actions?query=workflow%3ABuild)
+  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-postgresql.svg)](https://github.com/ballerina-platform/module-ballerinax-postgresql/commits/main)
+    [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/postgresql.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fpostgresql)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+  [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-postgresql/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-postgresql)
+
+The PostgreSQL library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+This provides the functionality required to access and manipulate data stored in a PostgreSQL database.  
+
+For more information on the operations supported by the `postgresql:Client`, which include the below, go to [The PostgreSQL Package](https://ballerina.io/learn/api-docs/ballerina/postgresql/).
+
+- Pooling connections
+- Querying data
+- Inserting data
+- Updating data
+- Deleting data
+- Updating data in batches
+- Executing stored procedures
+- Closing the client
+
+## Issues and Projects 
+
+Issues and Projects tabs are disabled for this repository as this is part of the Ballerina Standard Library. To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library). 
```

Comment:
```suggestion
Issues and Projects tabs are disabled for this repository as this is part of the Ballerina Standard Library. To report bugs, request new features, start new discussions, view project boards, etc., visit the Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library). 
```

Reply:
I updated. Commit id is fb00e99b1e6ee7ff1be5ede4529d1571294c214b